apps/etterlatte-behandling/src/main/kotlin/Contekst.kt

```diff
@@ -73,32 +66,27 @@ class SaksbehandlerMedEnheterOgRoller(
         return identifiedBy.hentTokenClaims(AZURE_ISSUER)!!.getStringClaim("NAVident")
     }
 
-    fun erSuperbruker() = name() in (saksbehandlereMedTilgangTilAlleEnheter)
-
-    fun enheter() =
-        if (saksbehandlerMedRoller.harRolleNasjonalTilgang()) {
-            Enheter.nasjonalTilgangEnheter()
-        } else {
-            try {
-                runBlocking {
-                    navAnsattKlient.hentEnhetForSaksbehandler(name()).map { it.id }
-                }
-            } catch (e: Exception) {
-                throw HentEnhetException("Henting av enheter feilet. Sjekk on-prem status(fss)", e)
+    fun enheterMedSkrivetilgang() =
+        try {
+            runBlocking {
+                navAnsattKlient.hentEnhetForSaksbehandler(name())
```

Comment:
Skal vi hente hver gang eller hente fra vår saksbehandlertabell?

Reply:
Godt spørs - jeg tar en prat med deg om det - kjenner ikke til hva du har laget der så godt :)

Reply:
Per nå ja - vi lager en jira for å ta seg av denne separat.

Reply:
EY-3537